test/observable/getObservable-test.js

```diff
@@ -6,15 +6,20 @@ var getObservable = require('../../lib/observable/getObservable');
 
 describe('getObservable', function() {
 	it('should return null for non-object', function() {
+		assert.same(null, getObservable(0));
 		assert.same(null, getObservable(1));
 		assert.same(null, getObservable(undefined));
+		assert.same(null, getObservable(null));
```

Comment:
Well, i guess `null` IS technically an object, so maybe I remove this assertion?